src/components/picker/index.tsx

```diff
@@ -224,7 +225,7 @@ const Picker = React.forwardRef((props: PickerProps, ref) => {
         useWheelPicker={useWheelPicker}
         mode={mode}
         useDialog={useDialog}
-        items={useItems ? items : undefined}
+        items={useItems ? filteredItems || items : undefined}
```

Comment:
IMO this should be `filteredItems` and not `filteredItems || items`, just like below there's only `{filteredChildren}`